routes/membership.js

```diff
@@ -19,9 +19,22 @@ const {
 const router = express.Router();
 
 router.get('/', checkAdmin, async (req, res, next) => {
-	const season = await req.db.Season.getCurrent();
+	const selectedSeason = req.query.season;
+	let season;
+	if (!selectedSeason) {
+		season = await req.db.Season.getCurrent();
+	}
+	else {
+		season = await req.db.Season.findOne({
+			where: {
+				id: req.query.season
+			}
+		});
+	}
 
-	if (!season) {
+	const seasons = await req.db.Season.findAll();
```

Comment:
We should probably sort the seasons here

Reply:
Okay yeah fair point